composer.json

```
@@ -2,17 +2,14 @@
     "name": "rubn/randy",
     "authors": [
         {
-            "name": "Ruben Zagon",
-            "email": "ruben.zagon@gmail.com"
+            "name": "Ruben Zagon y Carlos",
+            "email": "rubenzg.dev@gmail.com"
         }
     ],
```

authors es un array podrían ponerlo como dos elementos:
```suggestion
"authors": [
   {},
   {}
],
```